data-prepper-plugins/http-source/src/main/java/com/amazon/dataprepper/plugins/source/loghttp/HTTPSourceConfig.java

```diff
@@ -31,18 +32,24 @@
     static final int DEFAULT_MAX_PENDING_REQUESTS = 1024;
 
     @JsonProperty("port")
+    @Min(0)
+    @Max(65535)
     private int port = DEFAULT_PORT;
 
     @JsonProperty("request_timeout")
+    @Min(0)
```

Comment:
For all of these values that are supposed to be greater than or equal to 0, does `@Min(0)` allow for a zero value to be set?

Reply:
Yes, from the Javadocs:
> The annotated element must be a number whose value must be higher or equal to the specified minimum.